phpunit/class-wp-theme-json-test.php

```diff
@@ -4813,7 +4813,6 @@ public function test_get_top_level_background_image_styles() {
 							'background' => array(
 								'backgroundImage'    => array(
 									'url'    => 'http://example.org/image.png',
```

Comment:
Tiny nit: only needs a single space now:

```suggestion
									'url' => 'http://example.org/image.png',
```